pkgs/development/python-modules/buildbot/default.nix

```diff
@@ -80,7 +80,8 @@ let
     '';
 
     # TimeoutErrors on slow machines -> aarch64
-    doCheck = !stdenv.isAarch64;
+    # 2.8.0 has broken tests: https://github.com/buildbot/buildbot/pull/5322
+    doCheck = !stdenv.isAarch64 && version != "2.8.0";
```

Comment:
I would rather just checkout source if that's not too much of an issue. Or give upstream some time to issue a patch update

Reply:
I updated the derivation to use `fetchFromGitHub`, and linked to the PR above the call to it.